cmd/mesos_metrics_test.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import "testing"
+import "encoding/json"
+
+func parseon() string {
+	type Message struct {
+		Name string
+	}
+	var m Message
+	b := []byte(`{"Name":"Alice","Body":"Hello","Time":1294706395881547000}`)
+
+	json.Unmarshal(b, &m)
+	return m.Name
+}
+
+// TestMesosMetricsCheck getheiht
+func TestMesosMetricsCheck(t *testing.T) {
```

Comment:
This test is named TestMesosMetricsCheck but it's only testing URL generation.  I think it would be good to add more testing to verify actual runtime behavior.  You can do this by spinning up an `httptest.Server`. There are examples you can look at in the dcos-backup test suite.